test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -24,23 +24,26 @@ namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests
 {
     public class IntegrationTests
     {
-        private const string CollectorEndpointEnvVarName = "OTEL_EXPORTER_OTLP_ENDPOINT";
-        private static readonly string CollectorEndpoint = SkipUnlessEnvVarFoundFactAttribute.GetEnvironmentVariable(CollectorEndpointEnvVarName);
+        private const string CollectorEndpointEnvVarName = "OTEL_COLLECTOR_HOSTNAME";
```

Comment:
Curious, what's the different between endpoint and hostname? (my understanding is that TCP endpoint contains more information than a hostname - based on experience working with C++ Boost https://www.boost.org/doc/libs/1_47_0/doc/html/boost_asio/reference/ip__tcp/endpoint.html)

Reply:
That is exactly how I see it